### ISSUE TITLE
Create/Integrate a DEFNA Newsletter

### ISSUE THREAD

**Issue by sodevious** [DEFNA/old.defna.org#72]
Setup a newsletter signup form hosted on our defna site, I am assuming we can do something like this with Sendy? @adamfast

**Comment by jefftriplett** [comment]
I think we should give https://buttondown.com a try. They are Django based and are free to cheap to get started. We also don't have to have someone reboot the server.

**Comment by adamfast** [comment]
I agree buttondown is worth a shot - I looked a couple months ago and I believe they offer a nonprofit discount, and it works out to just about what our sendy hosting bill is. We haven't needed a reboot since we got off of the Digital Ocean droplet though.

I don't have the availability to attempt migrating until after the conference, though.

Sendy does offer a subscribe form (while logged in you can find it https://sendy.co/forum/discussion/18676/how-to-create-a-subscription-form#gsc.tab=0)

**Comment by jefftriplett** [comment]
They support multiple newsletters which can help us on the dcus front too.

**Comment by sodevious** [comment]
cool beans, down to tackle this project with adam sometime after the conf.
